encode_utils/tests/test_connection.py

```diff
@@ -18,7 +18,11 @@
 from encode_utils import connection
 from encode_utils import profiles
 
-DATA_DIR = "data"
```

Comment:
I actually have a fix for this data dir issue that I think would be preferred. It sets it in 
``encode_utils/tests/__init__.py``  to ```DATA_DIR = os.path.join(os.path.dirname(__file__),"data")````.

Reply:
yeah that was a brutal hack